src/component/form.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { useState } from 'react'
+import uuid from 'uuid'
+
+interface Props {
+  onSubmit: (todo: ToDo) => void
+}
+
+export function Form(props: Props) {
+  const initial = '';
+  const [value, setValue] = useState(initial);
+
+  const onChange = (event: any) => {
+    setValue(event.target.value)
+  };
+
+  const onSubmit = (event: any) => {
```

Comment:
👍

Reply:
I will check what type should be on event